fractions/keycloak/src/main/java/org/wildfly/swarm/keycloak/deployment/KeycloakAdapterConfigResolver.java

```diff
@@ -34,6 +34,7 @@ public static void setPathDeployments(Map<String, KeycloakDeployment> map) {
     @Override
     public KeycloakDeployment resolve(OIDCHttpFacade.Request request) {
         String path = request.getRelativePath();
-        return pathDeployments.get(path);
+        return pathDeployments.entrySet().stream().filter(e -> path.startsWith(e.getKey())).findFirst()
+            .map(e -> e.getValue()).orElseThrow(() -> new IllegalStateException("KeycloakDeployment is null"));
```

Comment:
Better error message? Something like `"No Keycloak configuration for URL " + path`?